data/shubhsharma19.json

```diff
@@ -67,6 +67,14 @@
     }
   ],
   "milestones": [
+    {
+      "title": "Got selected as Elite Mentor at ShowwcaseHQ",
+      "date": "March, 2023",
+      "icon": "FaGithub",
```

Comment:
Ig you have used the wrong icon 
and congratulations on your achievement :100:

Reply:
There is no showwcase icon that's why!

Reply:
Seems like it has nothing to do with GitHub. If you wanted to, then you can update it with something else from the [valid icons](https://linkfree.io/icons), else let us know to get in merged. 🙂